### ISSUE TITLE
[FEATURE] More lyrics alignment options

### ISSUE THREAD

**Issue by martinmake** [SwagLyrics/SwagLyrics-For-Spotify#2691]
It would be nice to have flags for different alignment options.
Maybe centering would be enough, I don't think that anyone would want to use right alignment.

**Comment by aadibajpai** [comment]
Indeed, alignment options would be interesting. However, I think it's out of scope for a cli program, because then we would need to be calculating terminal width and what not to decide on how to center it evenly. I think it's more appropriate for the GUI (which iirc has slightly better alignment already).

https://github.com/SwagLyrics/SwagLyricsGUI
